variables.tf

```diff
@@ -639,13 +639,21 @@ variable "external_secrets_irsa_policies" {
 variable "external_secrets_ssm_parameter_arns" {
   description = "List of Systems Manager Parameter ARNs that contain secrets to mount using External Secrets"
   type        = list(string)
-  default     = ["arn:aws:ssm:*:*:parameter/*"]
```

Comment:
In [other repos](https://github.com/terraform-aws-modules/terraform-aws-iam/blob/master/modules/iam-role-for-service-accounts-eks/variables.tf#L170) we have fairly permissive default policies, instead of restrictive.
What should the best practice be here?  Should we have a default permissive policy, and let the module user override that?  Or have a default "closed" policy and force the user to specify the permissions?